device/centec/x86_64-centec_e582_48x2q4z-r0/E582-48x2q4z/qos.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
we have unified qos.json.j2 now, can you check other platform?

Reply:
https://github.com/Azure/sonic-buildimage/blob/master/device/arista/x86_64-arista_7050_qx32/Arista-7050-QX32/qos.json.j2